dassh/correlations/properties_Na.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+from dassh.logged_class import LoggedClass
+
+class mat_from_corr(LoggedClass):
+    """
+    Correlation object for sodium properties
+    
+    Parameters
+    ----------
+    prop : str
+        Property name to be calculated
+        
+    Returns
+    ----------
+    float
+        Property value
+        
+    Notes
+    ----------
+    Reference correlations can be found in:
+    [1] J. K. Fink and L. Leibowitz, "Thermodynamic and 
+    Transport Properties of Sodium Liquid and Vapor," 
+    ANL-RE-95-2, Argonne National Laboratory, 1995.
+    """
+    
+    def __init__(self, prop):
+        LoggedClass.__init__(self, 0, f'dassh.correlations.properties_Na')
+        self.prop = prop
+    
+    def __call__(self, temperature):
+        return getattr(self, self.prop)(temperature)
+    
+    def density(self, T):
```

Comment:
Please, add the docstring at least for each public method, providing the identifier for the corresponding correlation in the referenced work [1].